tracing/tracing.go

```diff
@@ -25,35 +25,60 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
-// StartSpan starts child span in a context.
-func StartSpan(ctx context.Context, opName string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
+// StartSan starts child span in a context.
+func StartSpan(ctx context.Context, opName string, opts ...trace.SpanStartOption) (context.Context, *Span) {
```

Comment:
I thought we could have tracing.Span(ctx, prefix, name) but then it cannot take SpanStartOption...

Reply:
You are right Kaz. We can have a separate methods that pass either `StartSpanOption` or `names`. I am not sure if this is something we want.